app/src/main/java/com/codesoom/assignment/application/AuthenticationService.java

```diff
@@ -0,0 +1,57 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.errors.WrongPasswordException;
+import com.codesoom.assignment.utils.JwtUtil;
+import io.jsonwebtoken.Claims;
+import org.springframework.stereotype.Service;
+
+import java.util.Objects;
+
+/**
+ * 회원 인증 service 입니다.
```

Comment:
이 정도가 적절할 것 같습니다.
```suggestion
 * 회원 인증 기능을 제공합니다.
```

이렇게 해도 괜찮겠죠.
```suggestion
 * 회원 인증을 담당합니다.
```